Chap_Terms.tex

```diff
@@ -20,23 +20,30 @@ \chapter{PMIx Terms and Conventions}
+\item \declareterm{session} 
+refers to a set of resources assigned by the \ac{WLM} that has been 
+reserved for one or more users. 
+A session is identified by a \emph{session ID} that is  
+a unique within the scope of the governing \acp{WLM}.
+Historically, \ac{HPC} sessions have consisted of a static allocation of resources - i.e., a block of resources assigned to a user in response to a specific request and managed as a unified collection. However, this is changing in response to the growing use of dynamic programming models that require on-the-fly allocation and release of system resources. Accordingly, the term \emph{session} in this document refers to a potentially dynamic entity, perhaps comprised of resources accumulated as a result of multiple allocation requests that are managed as a single unit by the \ac{WLM}.
 
-\item \declareterm{namespace} refers to a character string value assigned by the \ac{RM} or launcher (e.g., \code{mpiexec}) to a \textit{job}. All \textit{applications} executed as part of that \textit{job} share the same \emph{namespace}. The \emph{namespace} assigned to each \emph{job} must be unique within the scope of the governing \ac{RM} and often is implemented as a string representation of a numerical job ID. The \emph{namespace} and \emph{job} terms will be used interchangeably throughout the document.
+\item \declareterm{job} refers to a set of one or more \emph{applications} executed as a single invocation by the user within a session with a unique identifier, the \emph{job ID}, assigned by the \ac{RM} or launcher. For example, the command line ``\textit{mpiexec -n 1 app1 : -n 2 app2}'' generates a single \ac{MPMD} job containing two applications. A user may execute multiple \emph{jobs} within a given session, either sequentially or concerrently.
 
-\item \declareterm{application} refers to a single executable (binary, script, etc.) member of a \emph{job}.
+\item \declareterm{namespace} refers to a character string value assigned by the \ac{RM} to a \textit{job}.  The requester of the \textit{job} (e.g., \code{mpiexec} or a \ac{PMIx} client) may request a specific \emph{namespace} value.  All \textit{applications} executed as part of that \textit{job} share the same \emph{namespace}. The \emph{namespace} assigned to each \emph{job} must be unique within the scope of the governing \ac{RM} and often is implemented as a string representation of the numerical emph{Job ID}. The \emph{namespace} and \emph{job} terms will be used interchangeably throughout the document.
+
```

Comment:
made the "normal" case the first sentence and then clarified that you can also request a specific namespace name for your job, but tried to downplay that since it is not central to what a namespace is or how they normally are identified by most systems.  Also wanted to get rid of a.k.a.'s and .i.e and .e.g. as much as possible.